backend/news-service/src/RouteHandler.js

```diff
@@ -31,10 +34,16 @@ function fillTimeSpan(request, response) {
 // /api/request/timespan
 function requestTimespan(request, response) {
   const { requestId, requestedResource } = request.body;
-  logger.debug(`Got a request for: ${requestedResource}`);
+  logger.debug(`Got a request for: ${JSON.stringify(requestedResource)}`);
   if (requestedResource) {
-    response.sendStatus(200);
+    requestedResource.from = new Date(requestedResource.from);
+    requestedResource.until = new Date(requestedResource.until);
+    if (requestedResource.until >= Date.now()) {
```

Comment:
Shouldn't it be allowed to request `Date.now()` and if so shouldn't it be:
```suggestion
     if (requestedResource.until > Date.now()) {
```

Reply:
Done